scalameter-core/src/main/scala/org/scalameter/picklers/Pickler.scala

```diff
@@ -0,0 +1,18 @@
+package org.scalameter.picklers
+
+import scala.annotation.implicitNotFound
+
+
+@implicitNotFound(msg ="""No Pickler available for ${T}. Please define yours or
+import org.scalameter.picklers.nopickler._ and use SerializationPersistor.""")
+abstract class Pickler[T] {
+  def pickle(x: T): Array[Byte]
+
+  def tryUnpickle(a: Array[Byte]): (T, Array[Byte])
```

Comment:
One thing has came to my mind. It would become problematic to handle `List[String]` properly, because we don't know how large is single string in list before deserializing (part) of it.
So that's the reason I would leave it as it is.
What do you think?